docs/api-ref.md

```diff
@@ -2683,10 +2682,59 @@ The schedule methods are based upon the endpoints for schedules in the REST API
 
 Source files: server/endpoint/schedules_endpoint.py
 
+#### schedule.add_to_schedule
+
+```py
+schedules.add_to_schedule(schedule_id)
+```
+
+Adds a `DataSourceItem` or a `WorkbookItem` to a schedule.
+
+
+REST API: [Add Data Source to Schedule](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_jobs_tasks_and_schedules.htm#add_data_source_to_schedule)
+
+
+
+**Parameters**
+
+Name  |  Description
+:--- | :---
+`schedule_id` | The Tableau-internal identifier of the schedule to add to. Can be retrieved from a `ScheduleItem`, e.g., by performing `schedules.get()`
```

Comment:
Slight edit suggested:  The identifier (`luid`) of the schedule to add the data source or workbook to.  
@bcantoni let me know if "luid" here is incorrect. Also, in the description we say "Adds a `DatasourceItem` or a `WorkbookItem` to a schedule.", but we looks like you can also add a flow?

Reply:
I`ve added the `FlowItem` to the method description.
If using "luid" is appropriate, I'll adpat the parameter description as well.

Reply:
Good catch adding FlowItem as well.

@kramantab I think to be consistent with the REST API documentation we can keep the more generic "id" usage here rather than "luid". That might be a future task if the REST API docs are moving that direction: updating TSC id vs luid usage.

Reply:
I've updated the description correspondingly.